client/src/components/Group/Group.js

```diff
@@ -42,15 +42,17 @@ function Group(props) {
   return (
     <Card style={{ margin: 16 }} onClick={props.onClick}>
       <Row justify="space-around" align="middle">
-        <Col className="group" flex="100px">
-          <Avatar src={props.group.avatarURL} size={64} />
+        <Col lg={4} md={4} sm={24} xs={24} className="group">
```

Comment:
Haven't had a chance to run this locally myself but nice seeing use of Ant responsive columns!